website/docs/sql/datatype/array.md

```diff
@@ -0,0 +1,84 @@
+# Array Type
+
+An array is a sequential collection of elements packed into a single SQL value.
+Arrays are useful to model dependent information with an inherent ordering in a single row – for example, time-series data from longitudinal studies, or embedding vectors from machine learning models. 
+
+Generally, array values are denoted in curly braces with elements separated by commas, like so:
```

Comment:
What means generally here? Where? See PostgreSQL documentation, they explain the syntax part better IMHO.

Reply:
(I agree with your remarks, about the leading "generally"; it's superfluous and confusing.)

See my comment above, could you perhaps point out what specifically you like about the PostgreSQL syntax documentation? 

The section on array input in the Postgres Docs refers to C struct initializers and has to introduce a generic format to accommodate `box[]` – I'm not sure that we should take it as a reference.

Reply:
Also if you leave out "Generally", what means "are denoted in curly braces"? Denoted in this documentation? Denoted in Hyper SQL? If the latter, what part of it? E>g., I cannot just use curly braces in SQL, so this definitly doesn't seem to be SQL syntax. I can use them in strings, but that should be called out as such.

The [PostgreSQL documentation](https://www.postgresql.org/docs/current/arrays.html#ARRAYS-INPUT) for example mentions exactly how an array string literal has to look like, including how values need to be escaped.  This is missing here if I see it correctly.

Reply:
I reworked the section: [bda7bea](https://github.com/tableau/hyper-db/pull/110/commits/bda7bea6e20cbb11681f1ca50140f8258c77a809)